Components/Alert/alert.css

```diff
@@ -0,0 +1,40 @@
+@import "../../style.css";
+
+.alert {
+  max-width: 20rem;
+  padding: 0.75rem 1rem;
+  margin: 0.5rem auto;
+  border-radius: 0.25rem;
+  border-width: 2px;
+  border-style: solid;
```

Comment:
You can use px for value smaller than 1rem as suggested. Also you can use the shorthand property for border-width and border-style. Rest is good.

Reply:
I used separate border properties so that in different button classes I'll have to change only the border colour, and not repeat the width, style again